File: LE6/d10-2.cpp
#include <iostream>
using namespace std;

void gerar_tabuada(int numero) {
    for (int i = 1; i <= 10; i++) {
        cout << numero << " x " << i << " = " << (numero * i) << endl;
    }
}

int main() {
    int numero;

    cout << "Digite um número: ";
    cin >> numero;

    gerar_tabuada(numero);

    return 0;
}

File: RP6/d10-2.java
import java.util.Scanner;

public class Ex10_V2 {

    public static void gerar_tabuada(int numero) {
        for (int i = 1; i <= 10; i++) {
            System.out.println(numero + " x " + i + " = " + (numero * i));
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int numero;
        System.out.print("Digite um número: ");
        numero = sc.nextInt();

        gerar_tabuada(numero);
    }
}